libraries/AP_HAL_ChibiOS/hwdef/Morakot/defaults.parm

```diff
@@ -0,0 +1,97 @@
+# Define ethernet settings

```

Comment:
unless you have a really good reason, these should be left at inherent defaults and this section removed.